src/app/components/pool/action/add/os-picker/os-image-picker/os-image-picker.component.spec.ts

```diff
@@ -80,7 +198,7 @@ describe("OSImagePickerComponent", () => {
 
     beforeEach(() => {
         poolOsServiceSpy = {
-            offers: new BehaviorSubject(skuSet1),
+            offers: of(skuSet1),
```

Comment:
Just wondering here, was there an issue with `new BehaivorSubject`? wasn't needed here because we don't change the value of offers for the test